### PR TITLE
Updating api-load to match latest release

### PR DESCRIPTION
**Pull request by chentex**
Signed-off-by: Vicente Zepeda Mas <vzepedam@redhat.com>

### Description

Updating api-load to match latest [release](https://github.com/cloud-bulldozer/ocm-api-load/releases/tag/v0.3.2)
- Removes extra step to upload to elastic since now it's done within
  ocm-api-load
- Adds flags to use elastic search indexing within ocm-api-load

### CODE REVIEW

**Comment by rsevilla87** [comment]
@chentex, API load tests passed, I think this is ready to merge.

File: e2e/api_load/api_load.yaml [hunk 1]
```diff
@@ -24,6 +24,7 @@ spec:
       aws_access_key: empty
       aws_access_secret: empty
       aws_account_id: empty
+      aes_region: empty
```
Comment (venkataanil) [inline e2e/api_load/api_load.yaml:27]: typo here

**Review by venkataanil — COMMENTED** [review]
LGTM

**Review by venkataanil — APPROVED** [review]

**Review by rsevilla87 — APPROVED** [review]
lgtm!

**Pull request merged**